virt_windows.go

```diff
@@ -0,0 +1,7 @@
+// +build windows
```

Comment:
purpose of this file?

Reply:
To fix cyclic dependencies

Reply:
It cannot be included neither `Cagent` not other files as it is windows specific dependency (in can be included actually into `processes_windows.go` if we don't want to have empty file)
Idea is VM provider is registered by `func init()`. This prevent instantiating inappropriate VM provider from within unsupported host system. To call that init we need somehow import VM provider package

Reply:
I see... nasty OS dependency 😅 
Please add a comment why we have this empty file. You can then resolve this conversation